<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetting</class>
 <widget class="QWidget" name="CameraSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(61, 61, 61, 0.6);
color: white;
border: 1px solid rgba(204, 0, 204, 0.6);
border-radius: 5px;       
</string>
   </property>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>340</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Дистанция камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Перемещение камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Вращение камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_5">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_6">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_7">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_8">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_9">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>300</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_14">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>скейл:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_x">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_y">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_z">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>230</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="rot_x">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>300</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="rot_y">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>300</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="rot_z">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>300</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>380</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QGroupBox" name="type_camera">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,0.6);
background-color: rgba(61, 61, 61, 0.0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="free">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(61, 61, 61, 0.6);
</string>
     </property>
     <property name="text">
      <string>Свободная</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="focus">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(61, 61, 61, 0.6);
</string>
     </property>
     <property name="text">
      <string>Фокусная</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_back_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Тип камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="type_projections">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,0.6);
background-color: rgba(61, 61, 61, 0.0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="paral">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(61, 61, 61, 0.6);
</string>
     </property>
     <property name="text">
      <string>Параллельная</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="persp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(61, 61, 61, 0.6);
</string>
     </property>
     <property name="text">
      <string>Переспективная</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_back_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Тип проекции</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_12">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>420</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Настройки видимости</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_13">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>дальний план:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_15">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>FOV:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_16">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>ближний план:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="far_plane">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200000.000000000000000</double>
    </property>
    <property name="value">
     <double>1500.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="near_plane">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>-9.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fov">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>30.000000000000000</double>
    </property>
    <property name="maximum">
     <double>120.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>45.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
